proto/data.go

```diff
@@ -677,3 +667,18 @@ func (ms *MVCCStats) Accumulate(oms MVCCStats) {
 	ms.GCBytesAge += oms.GCBytesAge
 	ms.LastUpdateNanos += oms.LastUpdateNanos
 }
+
+// Subtract subtracts the values of oms from ms.
+func (ms *MVCCStats) Subtract(oms *MVCCStats) {
```

Comment:
Accumulate and Subtract are named strangely.
Perhaps Add and Subtract, or Accumulate and Dissipate.